solcast/api.py

```
@@ -34,10 +27,13 @@ def __repr__(self):
     def to_dict(self):
         if self.code not in [200, 204]:
             raise Exception(self.exception)
-        if self.code == 204:  # Case of valid no content
+        if self.code == 204 or not self.data:  # Case of valid no content
```

Unexpected behaviour that would be better to raise an exception